### PR TITLE
Bugfix: Chat launcher icon added nav bar, previous launch button hidden

### PR DESCRIPTION
**Pull request by space-age-pete**
## Description

The chat feature will now be launched from an icon in the navbar rather than the floating button that has been blocking the run button

## Tests and performance

 - [X ] Changes to any .js files are covered by existing tests or this PR adds new tests (if not please explain why below)
 - [X] No new packages are added or package size and performance considerations are discussed below
 - [X] PR Title fits our [changelog format](https://github.com/PrefectHQ/ui#readme)

### CODE REVIEW

File: src/components/Nav/ApplicationNav.vue [hunk 1]
```diff
@@ -108,6 +110,8 @@ export default {
 
     <HelpMenu />
 
+    <ChatLauncher />
```
Comment (znicholasbrown) [inline src/components/Nav/ApplicationNav.vue:113]:
```suggestion
    <ChatLauncher v-if="isCloud" />
```

This should only be on the nav bar if we're in a Cloud context.

File: src/plugins/salesforceChat.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ export const initializeSalesforceChat = () => {
   const license = store.getters['license/license']
 
   const initESW = function(gslbBaseURL) {
-    window.embedded_svc.settings.displayHelpButton = true // Or false
+    window.embedded_svc.settings.displayHelpButton = false // Or false
```
Comment (znicholasbrown) [inline src/plugins/salesforceChat.js:18]:
```suggestion
    window.embedded_svc.settings.displayHelpButton = false
```
Probably no need for a comment here.

**Review by znicholasbrown — CHANGES_REQUESTED** [review]
A couple of small comments @space-age-pete

**Comment by zhen0** [comment]
Just checking - Are all the font-awesome file changes necessary?

**Comment by znicholasbrown** [comment]
@zhen0 - yeah those are necessary to add new icons to the project

**Review by znicholasbrown — APPROVED** [review]
LGTM @space-age-pete 

Closes: #1287

**Pull request merged**